app/sessionStore.js

```diff
@@ -254,12 +256,7 @@ module.exports.cleanAppData = (data, isShutdown) => {
   if (data.sites) {
     const clearHistory = isShutdown && getSetting(settings.SHUTDOWN_CLEAR_HISTORY) === true
     if (clearHistory) {
-      // TODO - this should the history methods from siteUtils
-      data.sites = data.sites.filter((site) => site && site.tags && site.tags.length)
-      data.sites = data.sites.map((site) => {
-        delete site.lastAccessedTime
-        return site
-      })
+      data.sites = siteUtil.clearHistory(Immutable.fromJS(data.sites)).toJS()
```

Comment:
@bridiver per your feedback, I updated this to use the siteUtil method :smile: 

I'm curious what you think about the temp conversion to/from Immutable. I originally made the siteUtil method detect if the input is Immutable, but if feels bad introducing more code paths.